common/lib/xmodule/xmodule/x_module.py

```diff
@@ -35,6 +35,7 @@
 from xmodule.exceptions import UndefinedContext
 import dogstats_wrapper as dog_stats_api
 
+from xmodule import course_metadata_utils
```

Comment:
Same comment about importing only the methods needed. Also, this import needs to be moved up.